src/Domain/Account/AccountId.php

```diff
@@ -10,4 +10,9 @@ public function __construct(
         public string $value
     ) {
     }
+
+    public function isEqualTo(AccountId $id): bool
```

Comment:
```suggestion
    public function isEqualTo(AccountId $id): bool
```

In Java the method name is `equals()`. Same is used in many popular libraries: https://github.com/ramsey/uuid/blob/4.x/src/UuidInterface.php#L64

Reply:
Agree. But in our case we use `brick/money` that [use](https://github.com/brick/money/blob/master/src/AbstractMoney.php#L138) `isEqualTo` name. `brick/money` is closer to this package.

Reply:
OK.